### ISSUE TITLE
MULTIPLE_BODY_PARAMETERS error for application/x-www-form-urlencoded

### ISSUE THREAD

**Issue by kevinoid** [Azure/oav#1057]
If an OpenAPI 2.0 document contains an operation which contains multiple parameters in formData and consumes `application/x-www-form-urlencoded`, validation fails with `MULTIPLE_BODY_PARAMETERS`.  If consumes is changed to `multipart/form-data`, validation succeeds without error.  For example:

```json
{
  "swagger": "2.0",
  "info": {
    "title": "Title",
    "version": "1.0"
  },
  "paths": {
    "/": {
      "post": {
        "operationId": "test",
        "consumes": [
          "application/x-www-form-urlencoded"
        ],
        "produces": [
          "application/json"
        ],
        "parameters": [
          {
            "name": "first_name",
            "in": "formData",
            "type": "string"
          },
          {
            "name": "last_name",
            "in": "formData",
            "type": "string"
          }
        ],
        "responses": {
          "204": {
            "description": "Example response"
          }
        }
      }
    }
  }
}
```

The OpenAPI document validates and Autorest generates correct code, but oav validation fails.

This issue is similar to #805.

I believe it could be fixed by checking for `application/x-www-form-urlencoded` in addition to `multipart/form-data` at
 
https://github.com/Azure/oav/blob/bf50f770da65ab3c1026a3a6744113c061cb97f4/lib/swaggerValidator/semanticValidator.ts#L527

Thanks for considering,
Kevin

**Comment by mikeharder** [comment]
I agree the rule should be updated to allow `application/x-www-form-urlencoded`.  From the spec:

> Form parameters are used to describe the payload of requests with `Content-Type` of:
> - `application/x-www-form-urlencoded` (used to POST primitive values and arrays of primitive values).
> - `multipart/form-data` (used to upload files or a combination of files and primitive data).

https://swagger.io/docs/specification/v2_0/describing-parameters/#form-parameters
